app/views/layouts/_topbar.html.erb

```diff
@@ -1,27 +1,27 @@
 <% if user_signed_in? %>
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <a class="navbar-brand" href="#">
-    <img src="https://www.gstatic.com/images/branding/product/1x/keep_48dp.png" class="d-inline-block align-top" width="40" alt="">
-    <span class="logo-title"> Keep</span>
-  </a>
-  
-  <form class="form-inline navbar-search">
-    <div class="input-group">
-      <div class="input-group-prepend">
-        <span class="input-group-text fas fa-search form-input-icon"></span>
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="/notes">
+      <img src="https://www.gstatic.com/images/branding/product/1x/keep_48dp.png" class="d-inline-block align-top" width="40" alt="">
```

Comment:
Don't use static images links from other sources, 
this can break in the future without knowing us.